src/wasmv1_execution/abi/abis.rs

```diff
@@ -1835,3 +1855,95 @@ pub fn abi_checked_div_native_time(
         },
     )
 }
+
+pub fn abi_date_now(
+    store_env: FunctionEnvMut<ABIEnv>,
+    arg_offset: i32,
+) -> Result<i32, WasmV1Error> {
+    handle_abi(
+        "date_now",
+        store_env,
+        arg_offset,
+        |handler,
+         _req: DateNowRequest|
+         -> Result<AbiResponse, WasmV1Error> {
+            let Ok(date) = handler.interface.date_now() else
```

Comment:
I would have used a `match` here (so we can forward the error to the caller)
+
error message is not correct ;)

Reply:
Done